cmp/layout/Tags.js

```diff
@@ -42,4 +42,5 @@ export const textarea = elemFactory('textarea');
 export const th = elemFactory('th');
 export const thead = elemFactory('thead');
 export const tr = elemFactory('tr');
-export const ul = elemFactory('ul');
\ No newline at end of file
+export const ul = elemFactory('ul');
+export const b = elemFactory('b');
```

Comment:
Nit - could we use `strong` instead, or `span({className: 'xh-bold'})`?  

(See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b re `b` as tag of last resort.)

Reply:
10-4